diff --git a/hal/Android.mk b/hal/Android.mk
index 91e497331..65088e1ac 100644
--- a/hal/Android.mk
+++ b/hal/Android.mk
@@ -327,7 +327,7 @@ LOCAL_SHARED_LIBRARIES := \
        libexpat
 
 ifneq ($(strip $(TARGET_USES_AOSP_FOR_AUDIO)),true)
-    LOCAL_SHARED_LIBRARIES += libtinycompress_vendor
+    LOCAL_SHARED_LIBRARIES += libtinycompress
 else
     LOCAL_SHARED_LIBRARIES += libtinycompress
 endif
